types/ConfigSchema.ts

```diff
@@ -1,15 +1,13 @@
 import { Cfgu } from './Cfgu';
 
-export type ConfigSchemaType = "json";
+export interface ConfigSchemaContentsValue extends Cfgu {};
+export type ConfigSchemaContents = { [ConfigKey: string]: ConfigSchemaContentsValue };
```

Comment:
I wonder if this can be simplified a little - something like:
```suggestion
export type ConfigSchemaContents = Record<string, Cfgu>;
```
Ideally not even having an auxillary type and typing `ConfigSchema.contents` directly as `Record<string, Cfgu>` would  be nice, but I'm not sure what to do then about `ConfigSchema.parse` in the `ts` sdk.

Reply:
I've pondered that as well. When initially integrating quicktype into the project, I created numerous tests. However, attempting to optimize any code in the types led to unsatisfactory generated results. Feel free to experiment with it and run `npm i` at the root dit afterward to observe the effects.

<img width="1374" alt="image" src="https://github.com/configu/configu/assets/11806915/9aa106ca-4d21-42db-aac6-15ab8bfb93e7">

<img width="1405" alt="image" src="https://github.com/configu/configu/assets/11806915/c2cb9dd2-c04a-466f-80b8-bee616d53d9e">